dashboards-observability/public/components/visualizations/plotly/plot.tsx

```diff
@@ -68,16 +72,23 @@ export function Plt(props: PltProps) {
   };
 
   return (
-    <PlotComponent
-      divId="explorerPlotComponent"
-      data={props.data}
-      style={{ width: '100%', height: props.height || '100%' }}
-      onHover={props.onHoverHandler}
-      onUnhover={props.onUnhoverHandler}
-      onClick={props.onClickHandler}
-      useResizeHandler
-      config={finalConfig}
-      layout={finalLayout}
-    />
+    <div>
+      <Annotations
+        showInputBox={props.showAnnotationInput!}
+        onTextChange={props.onChangeHandler!}
+        onAddAnnotation={props.onAddAnnotationHandler!}
+      />
+      <PlotComponent
+        divId="explorerPlotComponent"
+        data={props.data}
+        style={{ width: '100%', height: props.height || '100%' }}
```

Comment:
Please create custom class and use className here.

Reply:
This was old code however I'm not able to create custom class because of the props.height used..